tests/helpers/DoRequest72.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+
+namespace Behat\Mink\Tests\Driver;
+
+require_once __DIR__ . '/AbstractTestClient.php';
+
+abstract class DoRequestReturnObject extends AbstractTestClient
+{
+    protected function doRequest($request): object
```

Comment:
Maybe we can do this (adding `: object`) using `sed` replace approach I've talked about in https://github.com/minkphp/driver-testsuite/pull/38.